[api] fix runservice api docu via token (#10398)
openSUSE/open-build-service: docs/api/api/api.txt

POST /trigger/runservice

>   This call needs a token to identify user and package which shall be updated

Reviewer: ```suggestion
  This call needs a token to identify the user and package which shall be updated
```

  via a source service. See the cmd=create in the /person/<userid>/token route.

>   The token itself must be delivered as HTTP header:

Reviewer: ```suggestion
  The token itself must be delivered as a HTTP header:
```

    Authorization: Token <TOKEN_STRING>

Parameters

>   project: To be used together with "package" to identify the object. Must not be used

Reviewer: ```suggestion
  project: Must be used in conjunction with the "package" parameter to identify the service to trigger. Must not be used
```

Author: hm, it violates our style a bit. One can not check easily the existing parameters here anymore in the list when the "package" parameter is only inside of the text

Reviewer: But the `package:` parameter is still present in a comparable fashion to how it used to be there. I'm just wondering what the issue here is?

>            when the token is bound to a package.

Reviewer: ```suggestion
           when the token is bound to a specific package.
```

>   package: see project parameter.

Reviewer: What does this mean? That the package parameter must not be used in the same way the project parameter shall not be used when the token is bound to a package?

Author: You always need project and package parameter together. Or neither of them.

Author: (in future the project parameter alone may work though, but this is not implemented)

Reviewer: > You always need project and package parameter together. Or neither of them.

Ah, ok. That part was not clear to me

Reviewer: Just out of interest, what does happen if you pass the project & package parameters but the token is bound to a specific package?

Author: an error is reported IIRC
